extension/oidcextension/authenticator.go

```diff
@@ -116,7 +120,7 @@ func (o *oidcAuthenticator) Authenticate(ctx context.Context, headers map[string
 	return ctx, nil
 }
 
-func (o *oidcAuthenticator) Start(context.Context) error {
+func (o *oidcAuthenticator) Start(ctx context.Context, host component.Host) error {
```

Comment:
what is `host` and why was it added here?

Reply:
This changed because it needs to inherit from the component interface now. https://github.com/Zocdoc/opentelemetry-collector/blob/8b67a09d745d9429a1d21f136d7f4283d56dd9be/component/component.go#L27